package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "coffeescript": "~1.12.7",
     "gaze": "~1.1.2",
-    "jasmine-growl-reporter": "~0.2.0",
+    "jasmine-growl-reporter": "^2.0.0",
     "jasmine-reporters": "~1.0.0",
```

Comment:
Based off the rest of the package.json, I imagine the owner won't accept an auto-update package. You will likely need to get rid of the '^'